### DISCUSSION TITLE
Target application launcher jar

### DISCUSSION THREAD

**Discussion by punx120** [JaneWardSandy/eclipse-pde-partial-idea#90] [Q&A] [answered]
Hi,

Thank you for writing this plugin. When I create a eclipse application, it says "No target application launcherJar" specified. I'm not sure what it means. I'm guessing it has to do with Exclipe Target in Settings - but I'm not sure what to add.

Thank you

**Comment by JaneWardSandy** [comment, marked as answer]
See [setup target platform](https://github.com/JaneWardSandy/eclipse-pde-partial-idea/wiki/Quick-start#setup-target-platform-设置目标平台)

we should add at least one of target that was contains `org.eclipse.equinox.launcher_version.jar`
> Please make sure there is at least one RCP root to the target source to get the launcher Jar (org.eclipse.equinox.launcher_version.jar) to support launcher debugging

If `Launcher jar` was not specified, we don't know how to launch the `equinox` OSGI at all. So it was the problem.
<img width="1094" alt="image" src="https://user-images.githubusercontent.com/15345355/196019202-4e817144-c0d8-4c25-ba1c-4769b45c3ad9.png">

**Comment by punx120** [comment]
Thank you - yeah after installing eclipse locally and pointing to that install - this part worked better. When i launch the application though, it's complaining about bundles - which i believe is due to the project not being configured properly.
